src/components/dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import Card from '../../interfaces/card.interface'
+import File from '../../interfaces/table.interface'
+import { DashboardCard } from '../dashboardCard/DashboardCard'
+import { DashboardCardFooter } from '../dashboardCardFooter/DashboardCardFooter'
+import { DashboardTable } from '../dashboardTable/DashboardTable'
+import './Dashboard.scss'
+import data from './data.json'
+
+
+export const Dashboard = () => {
+  return (
+    <div className="dashboard">
+      <div className="dashboard__search">
+        <div className="dashboard__container">
+          <div className="rectangle-icon"></div>
```

Comment:
You should use a self-close tag when it's empty. 

```suggestion
          <div className="rectangle-icon"/>
```

Reply:
Ok I'll make the corrections!